<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1129</width>
    <height>812</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Internet Secure Chat</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_port">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>100</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>6000</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_chat">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>551</width>
      <height>731</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_address">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>70</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>vlbelintrocrypto.hevs.ch</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_03">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>70</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ADDRESS :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_04">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>100</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>PORT :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_01">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>541</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CHAT</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_connect">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>130</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CONNECT</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_02">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>531</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SERVER - CONNECTION</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_05">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>250</y>
      <width>531</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SERVER - MESSAGES</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_message">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>280</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_06">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>280</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>MESSAGE :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_07">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>400</y>
      <width>531</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SERVER - TASKS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_08">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>430</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TYPE :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_type">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>430</y>
      <width>431</width>
      <height>91</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>shift</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>vigenere</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>RSA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>hash</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>DifHel</string>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="sl_size">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>620</y>
      <width>431</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_09">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>620</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>SIZE :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_10">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>610</y>
      <width>431</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_send">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>650</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SEND</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_btn_encode">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>550</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>encode</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">btn_groupe_command</string>
    </attribute>
   </widget>
   <widget class="QLabel" name="lbl_11">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>550</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>COMMAND :</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="rd_btn_decode">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>580</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>decode</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">btn_groupe_command</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="btn_groupe_command"/>
 </buttongroups>
</ui>
